force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -237,12 +289,13 @@ public with sharing class FlowControlLoopsCollections {
      * @return A Set of unique strings.
      */
     public static Set<String> createStringSet(List<String> inputList) {
-        // Create a new Set of strings
+        
         Set<String> stringSet = new Set<String>();
 
-        //add strings to the set
+        stringSet.addALL(inputList);   //add strings to the set
+
+        return stringSet; 
 
-        return null; // Replace null with the variable you used to store the result
```

Comment:
Good work. You could again save a few lines if you pass a list to a Set constructor directly. Here is my solution. 

```
    public static Set<String> createStringSet(List<String> inputList) {
        return new Set<String>(inputList); // Replace null with the variable you used to store the result
    }
```